### DISCUSSION TITLE
CallBack in WebView2

### DISCUSSION THREAD

**Discussion by mrBugSlayer** [MicrosoftEdge/WebView2Feedback#3969] [Q&A]
So the scenario is, I have one Login URL that provide callback when I hit Login Button in webform, and I will receive that callback in a particular method.
Same thing is happening in CefSharp by using JavaScriptObjectBinding but I am trying the same thing in WebView2 by using AddHostObjectToScript  but this not working for me.

Is there anyone who can provide me a solution?

**Comment by david-risney** [comment]
Would this be the same sort of issue described by https://github.com/MicrosoftEdge/WebView2Feedback/issues/2632 ? If so, that thread has a workaround and I've filed a bug to track addressing that issue.
